src/server/sql/reading/create_reading_views.sql

```diff
@@ -344,8 +240,6 @@ hourly_readings_unit
 
 
 -- TODO Check if needed and when to use as not done for hourly.
```

Comment:
Now that the daily view depends on the hourly view, the hourly view must be declared first. This is why all the tests could not run on the PR. I moved this below hourly and the basic problem went away but there are still failed tests (see other comment).